packages/text-input/src/TextInput.tsx

```diff
@@ -286,10 +286,10 @@ const TextInput = React.forwardRef(
         {description && <p className={descriptionStyle}>{description}</p>}
         <div className={inputContainerStyle}>
           <input
+            type="text"
```

Comment:
Could we make this the default value in the destructuring assignment, and then pass type explicitly?

Reply:
I'd also love for us to add explicit typing to allow for just text input types. I think they're:
`email`, `password`, `search`, `text`, `url`, `tel`. I could be missing one...

Reply:
🤦 makes sense -- updated!